code/test/sanityCheck.py

```diff
@@ -24,28 +35,29 @@ def checkRxnAnnotation(rxns):
     """
     check consistency of rxn lists between model and annotation file
     """
-    rxnAssoc = pd.read_table("model/reactions.tsv")
-    rxnList = rxnAssoc['rxns'].to_list()
+    rxnList = get_column_from_tsv("model/reactions.tsv", "rxns")
+    spontaneous = get_column_from_tsv("model/reactions.tsv", "spontaneous", False)
+    rxnDeprecated = get_column_from_tsv("data/deprecatedIdentifiers/deprecatedReactions.tsv", "rxns")
+    assert not bool(set(rxnList) & set(rxnDeprecated)), "Deprecated reaction reused!"
```

Comment:
I think it reads easier:

```suggestion
    assert set(rxnList).isdisjoint(set(rxnDeprecated)), "Deprecated reaction reused!"
```